src/MOI/MOI_wrapper.jl

```diff
@@ -1278,6 +1287,155 @@ function MOI.modify(
     return
 end
 
+function MOI.set(
+    model::Optimizer,
+    ::MOI.ObjectiveFunction{MOI.ScalarNonlinearFunction},
+    f::MOI.ScalarNonlinearFunction,
+)
+    parsed, type, value = Cint(1), Cint[], Cdouble[]
+    _reverse_polish(model, f, type, value)
+    reverse!(type)
+    reverse!(value)
+    push!(type, Lib.XPRS_TOK_EOF)
+    push!(value, 0.0)
+    @show type, value
+    @checked Lib.XPRSnlpdelobjformula(model.inner)
```

Comment:
@joaquimg, where did
https://github.com/jump-dev/Xpress.jl/blob/33737734c64fc08606d6ddc3581fd897cb58539c/src/lib.jl#L2289-L2291
come from? It's not in `xprs.h`

Reply:
There's a symbol for it, but no documentation:
```
(base) oscar@Oscars-MBP Xpress % nm /Applications/FICO\ Xpress/xpressmp/lib/libxprs.dylib | grep XPRSnlpdelobjformula
0000000000229df0 T _XPRSnlpdelobjformula
000000000022a090 T _XPRSnlpdelobjformula_unchecked
```

Reply:
I think we need the 9.0+ header for this.

Reply:
it is not in 8.5, nor 8.14 not 9.3.
Maybe its something that was removed from the main API

Reply:
I have 42.01.05

Reply:
<img width="1307" alt="image" src="https://github.com/jump-dev/Xpress.jl/assets/8177701/4e8bc4f4-dac2-4bb8-894c-12536c430aa9">

Web has nothing

Reply:
I think I found it in 9.0.2. : https://github.com/jump-dev/Xpress.jl/commit/549073a52bd7112ebe0e50c2e5fb54580d1ea14a (see commit name)
So it was probably deprecated

Reply:
Cool cool. Testing with v8 has made this much easier to avoid breakage.